docs/mine-hnt/validators/validator-deployment-guide.mdx

```diff
@@ -194,23 +186,65 @@ docker run -d \
+
+Additional flags that may be helpful:
+
+- `-e "NAT_INTERNAL_IP=192.168.0.139"` If your host machine is behind a firewall that is performing network address translation (NAT), adding this flag will assist the Validator to quickly determine the IP address of the host machine. Replace the IP with yours from `ip addr`. This goes hand-in-hand with the following flag:
+- `-e "NAT_EXTERNAL_IP=$( curl -s ipv4.icanhazip.com )"` This also assists the Validator in quickly identifying the external IP of the NAT. The `curl` commmand simply queries an external provider and returns the public IP address of the interface. You can remove this and replace with your external IP if it is known.
+- Similar commands are available with `NAT_INTERNAL_PORT` and `NAT_EXTERNAL_PORT` if any port translations are occuring. If not, these may be omitted.
+
+Once you run the command above, docker will retrieve the latest Validator image, and start the docker process. Barring any errors, your Validator is running.
+
+#### Configure your Docker to update the Validator automatically - RECOMMENDED
+
+Having all Testnet Validators running the latest release of Validator software is crucial to quickly test code additions, changes, and bug fixes. Having an out-of-date Validators can increase election times or even stall the chain depending on the changes made.
+
+To ensure that each Validator gets updated quickly without your manual involvement, we will use a docker container process called [Watchtower](https://containrrr.dev/watchtower/). This is a docker container that, based on an interval, reviews the running containers to see if there is an updated image available for the container. If so, it stops the running container, and creates a new container with the updated image.
+
+```
+docker run -d \
+--restart always \
+--name watchtower \
+-v /var/run/docker.sock:/var/run/docker.sock \
+containrrr/watchtower \
+--cleanup \
+--interval 1800
+```
+
+- `-v /var/run/docker.sock:/var/run/docker.sock` A volume flag to allow the watchtower container to monitor running containers.
+- `containrrr/watchtower` The Image of the watchtower container
+- `--cleanup` Passes a flag into the watchtower container. This instructs watchtower to remove old images after updating.
+- `--interval 1800` Poll interval (in seconds). This value controls how frequently watchtower will poll for new images. For Testnet, the recommendation is to check for an updated image every 30 minutes.
+
+Additional flags that may be helpful:
 
+- `quay.io/team-helium/validator:latest-val-amd64` Include the image text as the last line in the command above. Including this at the end simply tells watchtower to only monitor the validator image and not an of the other images in your docker server. This is included here for convenience if there are other docker containers running on the host. If the validator is the only service running, this can be ommitted.
```

Comment:
```suggestion
- `quay.io/team-helium/validator:latest-val-amd64` Include the image text as the last line in the command above. Including this at the end simply tells watchtower to only monitor the validator image and not any of the other images in your docker server. This is included here for convenience if there are other docker containers running on the host. If the validator is the only service running, this can be omitted.
```